samples/meetings-transcription/csharp/README.md

```diff
@@ -0,0 +1,147 @@
+﻿---
+page_type: sample
+description: This is a sample app with capability to send notification when user creates workitem in [Azure DevOps](https://dev.azure.com) via webhooks.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+contentType: samples
+createdDate: "10-06-2022 00:02:15"
+---
+
+# Bot to show the transcript of the meeting using Microsoft Graph API.
+
+Bot to show the transcript of the meeting using Microsoft Graph API.
+
+## Key features
```

Comment:
You should add detailed description for the functionality.

Reply:
Updated.